### DISCUSSION TITLE
How may tabs do I have open?

### DISCUSSION THREAD

**Discussion by gmorph** [piroor/treestyletab#3068] [Q&A] [answered]
I bet this is somewhere obvious and I've missed it but is there a way to tell how many tabs you have open in total?

Thanks, Grant.

**Comment by silverwings15** [comment, marked as answer]
you can check out some of the available styles here: https://github.com/piroor/treestyletab/wiki/Code-snippets-for-custom-style-rules#tab-numbering-and-counting. there are also many addons that contain this feature and more
